app/controllers/interactive_pages_controller.rb

```diff
@@ -179,12 +179,16 @@ def edit_embeddable_redirect(embeddable)
         param = { :edit_embed_lb => embeddable.id }
       when Embeddable::Xhtml
         param = { :edit_embed_xhtml => embeddable.id }
+      when Embeddable::ExternalScript
```

Comment:
Is this added so the item will be expanded in the accordion automatically?

Based on the comment on line 193 this should have caused these parameters should have caused the edit dialog to open automatically after the embeddable was added. But that doesn't happen.

@dougmartin you should know about this since you will be working on the popups. 

I think the code that is supposed to handle this is here:
https://github.com/concord-consortium/lara/blob/5e9343a9c0a9163f5cfacc897b448cfcbbba22ed/app/views/shared/_embedded_editor_links.html.haml#L64
And it looks like it is broken because it is expecting the id to be `"#{type}-#{embeddable.id}"` but the id sent above doesn't include the `#{type}-`

It looks like it was broken 5 years ago and we never caught it:
https://github.com/concord-consortium/lara/commit/4658516aa94fee2bb9dc4636c331c66f0965c2cc

That code was checking for the embeddable id here: https://github.com/concord-consortium/lara/commit/4658516aa94fee2bb9dc4636c331c66f0965c2cc#diff-7c27f2754a64715165c8e351a0e65349L8

But it was changed to look for the dash-id here:
https://github.com/concord-consortium/lara/commit/4658516aa94fee2bb9dc4636c331c66f0965c2cc#diff-e5acde65bdae3c0eeb5f79ed8be98d54R25

I'm going to ask about this on slack to see what authors prefer. If they prefer the current behavior of not automatically opening the dialog we should remove this code.

Reply:
@scytacki Before I added those lines, an edit_[type] param was appended to the URL only when you added certain assessment item types. I added those lines so that an edit param was appended whenever you add _any_ assessment item type. That way I could then check for the presence of an edit param and open the item that was just added when the page reloads.

Reply:
Yeah that is what I figured after digging into this more.  I added a separate tory about fixing the popup behavior: https://www.pivotaltracker.com/story/show/172711110

I asked Doug to work on it,  so we can get it onto staging as we revise the plan for popup authoring.